src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -75,4 +75,11 @@ private static boolean isNotEmpty(String toTest, String qualifiedFieldName, Erro
         }
         return true;
     }
+
+    public static void checkisEmpty(String toTest, String qualifiedFieldName, Errors errs, String loggingContext) {
```

Comment:
the 'checkisEmpty' name should have a capital 'I' for 'is'

Reply:
Corrected.